package/build/LibSassBuilder.props

```diff
@@ -3,7 +3,7 @@
   <Import Project="$(MSBuildThisFileDirectory)\DesignTime\LibSassBuilder.DesignTime.targets" Condition="'$(DesignTimeBuild)' == 'true'" />
 
   <PropertyGroup>
-    <SassExe Condition=" '$(SassExe)'=='' ">$(MSBuildThisFileDirectory)../tool/LibSassBuilder.dll</SassExe>
+    <SassExe Condition=" '$(SassExe)'=='' ">$(MSBuildThisFileDirectory)../tool/LibSassBuilder.exe</SassExe>
```

Comment:
I think it does, I'll spin up an ubuntu machine and tell you

Reply:
Whoops, sorry, I made a fix.

I tested this:
```bash
git clone https://github.com/JelleHissink/LibSassBuilder.git
cd LibSassBuilder/src
dotnet build
LibSassBuilder/bin/Debug/net6.0/LibSassBuilder
```

Outputs in docke on ubuntu in sdk/6.0:
```bash
Sass compile directory: /root/LibSassBuilder/src
Sass files compiled
```

Outputs on windows:
```bash
Sass compile directory: C:\Temp\xy\LibSassBuilder\src
Sass files compiled
```

Reply:
Oh wow I didn't know omitting the extension would work?! Can you please give some more context on this?

Happy to merge 👍

Reply:
On linux the executable name is without .dll or .exe.
On windows an executable is selected by finding the file or the file with the suffix .exe. You effectively had the same by calling dotnet, that actually starts dotnet.exe.
So in both cases this notation without extension works. 

The main problem was that on windows if there was a dotnet call, it could either start a x86 or x64 dotnet.exe process that loaded the LibSassbuilder, if somehow you ended up with the x86 version of dotnet.exe then the native library references for the sass engine would not be available.